README.md

```diff
@@ -12,13 +12,15 @@
 <tr><th>
 Browsers
 </th><td width=100%>
-
-Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.pika.dev](https://cdn.pika.dev)
-
+  
+Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
+ 
 ```html
 <script type="module">
-  import { Octokit } from "https://cdn.pika.dev/@octokit/core";
-  import { restEndpointMethods } from "https://cdn.pika.dev/@octokit/plugin-rest-endpoint-methods";
+  
+import { Octokit } from "https://cdn.skypack.dev/@octokit/core";
+import { restEndpointMethods } from "https://cdn.skypack.dev/@octokit/plugin-rest-endpoint-methods";
+  
```

Comment:
The changes look good, but you changed the formatting, could you revert these changes? 

- on line 15 & 17 you added spaces
- you added new empty lines 20 & 23
- you removed the spaces before the `import` statements on line 20/21

we call these "whitespace" changes. Could you revert these, please? Let me know if you have any questions

Reply:
Okay, I am creating a new pull request with the above changes .

Reply:
No need to create a new pull request, you can make the changes in your existing pull requests, directly in the browser. Open the [files changed](https://github.com/octokit/plugin-rest-endpoint-methods.js/pull/265/files) tab, click on the 3 dots on the top right and then on "edit this file" 

<img width="357" alt="image" src="https://user-images.githubusercontent.com/39992/97343984-64ed1080-1845-11eb-94de-1fcda2ee23e3.png">

You can make the changes right in there and commit your changes when you are done

Reply:
Actually, I am having some problem to edit in this pull request as it is requesting me to create a new branch for committing the changes, that's why I have created a new PR.

Reply:
Ah I see. The problem might be that GitHub cannot find the fork it created when you edited the file in the first place. That's why it says "unknown repository" 

<img width="180" alt="image" src="https://user-images.githubusercontent.com/39992/97347201-ab446e80-1849-11eb-935c-513cfbc90717.png">

Let me close this pull request and you try again, ok?

Reply:
Yes, sure !